api_proxy.yaml

```diff
@@ -193,8 +193,7 @@ paths:
         in: path
         required: true
         description: The ID of the message.
-        schema:
-          $ref: "#/definitions/InstallationID"
+        type: string
```

Comment:
if you add min-length: 1 will produce a NonEmptyString

Reply:
this don't cause the generation of any type, is itn't?

Reply:
yes, actually types are only generated for schemas under the "definitions" swagger section (not for parameters)